File: Question2.cpp
#include <iostream>
using namespace std;

int main() {
    int number;
    cout << "Please enter a number between 3 and 9 (inclusive): ";
    cin >> number;

    if (number < 3 || number > 9) {
        cout << "Invalid input! Please enter a number between 3 and 9." << endl;
        return 0;
    }
    for (int i = 1; i <= number; i++) {
        for (int j = 1; j <= i; j++) {
            cout << "*";
        }
        cout << endl;
    }
    for (int i = number - 1; i >= 1; i--) {
        for (int j = 1; j <= i; j++) {
            cout << "*";
        }
        cout << endl;
    }
    return 0;
}

File: Question2.py
num = int(input("Please enter a number between 3 and 9 (inclusive): "))
if num < 3 or num > 9:
    print("Invalid input! Please enter a number between 3 and 9.")
else:

    for i in range(1, num + 1):
        print("*" * i)
    for i in range(num - 1, 0, -1):
        print("*" * i)
